Cesium3DTilesSelection/src/TilesetContentManager.cpp

```diff
@@ -1118,6 +1152,18 @@ void TilesetContentManager::updateContentLoadedState(
   if (content.isExternalContent()) {
     // if tile is external tileset, then it will be refined no matter what
     tile.setUnconditionallyRefine();
+    tile.setState(TileLoadState::Done);
+  } else if (content.isRenderContent()) {
+    TileRenderContent* pRenderContent = content.getRenderContent();
+    if (pRenderContent) {
+      // add copyright
+      pRenderContent->setCredits(GltfUtilities::parseGltfCopyright(
+          *_externals.pCreditSystem,
+          pRenderContent->getModel(),
+          tilesetOptions.showCreditsOnScreen));
+
+      tile.setState(TileLoadState::CreatingResources);
```

Comment:
I think this new state isn't needed at all. Just leave the tile in the `ContentCreated` state until we're ready to create resources, and then do this `setCredits` bit plus the `createRenderResources` before transitioning to `Done`. No reason AFAICT that setCredits needs to happen earlier.

Reply:
This was a good idea and cleaned up the logic a good bit!